charts/meilisearch/README.md

```diff
@@ -108,6 +108,8 @@ helm uninstall <your-service-name>
 | | |
 | `volumeMounts`                   | Additional volumes to mount on pod                             | `[]`
 | | |
+| `containers`                     | Additional containers for pod                                  | `[]`
```

Comment:
Wouldn't it be better to name it `sidecar-container` or something more explicit? And shouldn't we put it in resources like: `resources.containers` or  `resources.sidecar-containers`?

Reply:
This could be prefixed, but this chart does not use any prefixed naming for other resources like volume mounts, which is also extra. Nesting it under resources could do more harm, as it is used to define compute resources rather than extra stuff.